lib/localparticipant.js

```diff
@@ -681,6 +692,19 @@ class LocalParticipant extends Participant {
  * @event LocalParticipant#trackStopped
  */
 
+/**
+ * One of the {@link LocalParticipant}'s {@link LocalTrack}s encounters a {@link TwilioWarning}.
+ * @param {TwilioWarning} name - The {@link TwilioWarning} that was raised.
+ * @param {LocalTrack} track - The {@link LocalTrack} that encountered the {@link TwilioWarning}.
```

Comment:
Let this be the LocalTrackPublication, since this is the object associated with the Room, whereas LocalTrack is global, and can be used in multiple Rooms.